stl/inc/yvals_core.h

```diff
@@ -866,7 +866,19 @@
 #define _DEPRECATE_EXPERIMENTAL_ERASE
 #endif // ^^^ warning disabled ^^^
 
-// next warning number: STL4027
+// P0768R1 [depr.relops]
+#if _HAS_CXX20 && !defined(_SILENCE_CXX20_REL_OPS_DEPRECATION_WARNING) \
+    && !defined(_SILENCE_ALL_CXX20_DEPRECATION_WARNINGS)
+#define _DEPRECATE_CXX20_REL_OPS                                                              \
+    [[deprecated("warning STL4027: "                                                          \
+                 "std::rel_ops is deprecated in C++20. It has been superseded by <compare>. " \

```

Comment:
I think "The namespace std::rel_ops and its contents are deprecated in C++20." is a bit more precise. Which breaks "it" in the second sentence: "Their use is superseded by C++20's <=> operator and automatic rewrites of relational expressions."?

Reply:
I originally had "superseded by operator<=>" originally. However, since that requires `<compare>` I thought it was better to include that as if you were to update to use `<=>` you would get errors about not including that header.

Reply:
`rel_ops` also supported `x != y`, which has nothing to do with `<compare>` or `operator<=>` after WG21-P1185. You need neither the header nor `<=>` to have the compiler rewrite `!=` expressions into `==`. This:
```c++
struct S {
  friend bool operator==(S, S) { return true; }
};

int main() {
  return S{} != S{};
}
```
is a valid C++20 program. "superseded by `<compare>` in conjunction with `operator<=>`" is misleading to a user who was using `rel_ops` only for `!=`. Conversely, I think it's less useful to mention `<compare>` here: if people go looking for something like `rel_ops` in `<compare>` they will be wasting time.

Reply:
`rel_ops` also supported `x != y`, which has nothing to do with `<compare>` or `operator<=>` after WG21-P1185. You need neither the header nor `<=>` to have the compiler rewrite `!=` expressions into `==`. This:
```c++
struct S {
  friend bool operator==(S, S) { return true; }
};

int main() {
  return S{} != S{};
}
```
is a valid C++20 program. "superseded by `<compare>` in conjunction with `operator<=>`" is misleading to a user who was using `rel_ops` only for `!=`. Conversely, I think it's less useful to mention `<compare>` here: if people go looking for something like `rel_ops` in `<compare>` they will be wasting time.

Reply:
`rel_ops` also supported `x != y`, which has nothing to do with `<compare>` or `operator<=>` after WG21-P1185. You need neither the header nor `<=>` to have the compiler rewrite `!=` expressions into `==`. This:
```c++
struct S {
  friend bool operator==(S, S) { return true; }
};

int main() {
  return S{} != S{};
}
```
is a valid C++20 program. "superseded by `<compare>` in conjunction with `operator<=>`" is misleading to a user who was using `rel_ops` only for `!=`. Conversely, I think it's less useful to mention `<compare>` here: if people go looking for something like `rel_ops` in `<compare>` they will be wasting time.

Reply:
Makes sense, I hadn't thought of the equality operators.